### ISSUE TITLE
Unable to connect to RDS Instance using password mode as askUser

### ISSUE THREAD

**Issue by voodoodror** [dbgate/dbgate#974]
**Describe the bug**
Titles say it all. I cannot login to an RDS MySQL database with SSL enabled when password mode is configured as `askUser`.
This doesn't happen when "Save and encrypt" option is selected (everything's working properly). I'm not sure whether RDS/SSL enabled has anything to do with it.
I can confirm that "Testing" button works, regardless of which password mode is selected.

**To Reproduce**
1. Go to DBgate
2. Add new connection, fill in hostname, select "Don't save, ask for login and password" on password mode
3. Press "Connect" results in "Testing connection" hangs forever

**Expected behavior**
I expect it to establish a database connection.

**Screenshots**
![image](https://github.com/user-attachments/assets/414a863f-24c0-4649-adfc-497150eef030)

I see the following errors on container logs (not sure it's related though):

```
{"pid":57,"caller":"apiIndex","name":"connect","errorMessage":"Cannot destructure property 'msgtype' of 'e' as it is undefined.","errorObject":{},"errorStack":"TypeError: Cannot destructure property 'msgtype' of 'e' as it is undefined.\n    at handleProcessCommunication (/home/dbgate-docker/bundle.js:2:1318894)\n    at process.<anonymous> (/home/dbgate-docker/bundle.js:2:1245187)\n    at process.emit (node:events:514:28)\n    at emit (node:internal/child_process:937:14)\n    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)","level":60,"msg":"Uncaught exception, exiting process","time":1735141939105}
{"pid":1,"caller":"useController","name":"main","errorMessage":"[object Object]","errorObject":{"detail":{"conid":"d96b0802-a386-41aa-a737-af315283d8b8","passwordMode":"askUser"}},"level":50,"msg":"Error when processing route /server-connections/listDatabases","time":1735141941489}
{"pid":1,"caller":"useController","name":"main","errorMessage":"[object Object]","errorObject":{"detail":{"conid":"d96b0802-a386-41aa-a737-af315283d8b8","passwordMode":"askUser"}},"level":50,"msg":"Error when processing route /server-connections/ping","time":1735141946091}
{"pid":1,"caller":"useController","name":"main","errorMessage":"[object Object]","errorObject":{"detail":{"conid":"d96b0802-a386-41aa-a737-af315283d8b8","passwordMode":"askUser"}},"level":50,"msg":"Error when processing route /server-connections/ping","time":1735141986097}
{"pid":1,"caller":"useController","name":"main","errorMessage":"[object Object]","errorObject":{"detail":{"conid":"d96b0802-a386-41aa-a737-af315283d8b8","passwordMode":"askUser"}},"level":50,"msg":"Error when processing route /server-connections/ping","time":1735142026394}

```

**Version Information (please complete the following information):**
 - OS: Mac
 - App Version: 6.1.1
 - Install source: Docker, bug confirmed on Mac's desktop app too
 - Type: Web/Application (both)
 - Database engine: MySQL

**Comment by voodoodror** [comment]
I can confirm that this bug applies to 6.1.0 versions and above
6.0.0 works good as expected

**Comment by voodoodror** [comment]
I can confirm that this bug applies to 6.1.0 versions and above
6.0.0 works as expected

**Comment by janproch** [comment]
fixed in 6.1.2
